pyveda/frameworks/batch_generator.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+from random import randint
+from random import sample
```

Comment:
this can be one line `from random import randint, sample`

Reply:
I'm actually going to remove both of those since they aren't used right now. They were used with the augmentation functions which are gone for now.